helpers/types.ts

```diff
@@ -13,6 +13,10 @@ export type HostMessage =
   | {
       type: "field-config";
       data: Config;
+    }
+  | {
+      type: "mode";
+      data: "view" | "edit";
```

Comment:
Maybe we can move `"view" | "edit"` into another type named `Mode`. We're using this on line 31 too.